guides/common/modules/proc_filter-errata-by-type-severity.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Filter_errata_by_type_or_severity_{context}"]
+= Filter Errata by Type or Severity
+
+Use the following procedure to filter errata by type or severity.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts > All Hosts* and click the name of the host.
+. In the new hosts page click the *Contents* tab.
```

Comment:
Same as above. We generally do not state which page the user is on. Please reword along with line 9.